dashboard/app/helpers/pd/workshop_survey_results_helper.rb

```diff
@@ -12,17 +12,20 @@ module Pd::WorkshopSurveyResultsHelper
   #
   # @param surveys List of TeacherconSurveys or LocalWorkshopSurveys
   # @param facilitator Facilitator name to restrict responses for
+  # @param facilitator_breakdown Whether to have a facilitator breakdown
   # @returns Hash representing an average of all the respones, or array of free text responses
-  def summarize_workshop_surveys(surveys, facilitator = nil)
+  def summarize_workshop_surveys(workshops, facilitator = nil, facilitator_breakdown = true)
```

Comment:
Suggestion: turn the optional parameters into keyword arguments (`facilitator: nil, facilitator_breakdown: true`), so they can be specified more clearly and only as needed.